site/content/docs/09-building.md

```diff
@@ -14,6 +14,14 @@ The output is a Node app that you can run from the project root:
 node __sapper__/build
 ```
 
+If you're deploying your app to a different machine than the one you're building on, make sure that you install your production dependencies (required by `server.js`) by installing all dependencies from `package.json` using `--only=prod` (or `--only=production`):
```

Comment:
We should remove this so as not to duplicate content from the deploying section

Reply:
Thanks for the reminder, I was still unsure back then but I think the wording in `11-deployment` states it more clearly, will remove this then. But before that, is it better if it was replaced with something like this?
> See [deployment section (link)] for instructions on deploying to a different machine

Reply:
No, I don't think it's necessary and would leave it out

Reply:
Okay then, thanks for the review